### ISSUE TITLE
develop: Rare I get a Fatal Error on Login

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#2031]
I got this on the first attempt to login. A repeated Login succeeded. 

In this case I had the credentials not for the server stored.

But I can't reproduce this.

```
Fatal error in MSS 8.3.0 on Linux-6.2.0-10018-tuxedo-x86_64-with-glibc2.35
Python 3.10.8 | packaged by conda-forge | (main, Nov 22 2022, 08:23:14) [GCC 10.4.0]

Please report bugs in MSS to https://github.com/Open-MSS/MSS

Information about the fatal error:

Traceback (most recent call last):
  File "/home/reimar/MAIN/MSS/mslib/msui/mscolab.py", line 326, in login_handler
    self.save_user_credentials_to_config_file(data["email"], data["password"])
  File "/home/reimar/MAIN/MSS/mslib/msui/mscolab.py", line 331, in save_user_credentials_to_config_file
    save_password_to_keyring(service_name=self.mscolab_server_url, username=emailid, password=password)
  File "/home/reimar/MAIN/MSS/mslib/utils/auth.py", line 89, in save_password_to_keyring
    keyring.set_password(service_name=service_name, username=username, password=password)
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/keyring/core.py", line 61, in set_password
    get_keyring().set_password(service_name, username, password)
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/keyring/backends/SecretService.py", line 91, in set_password
    collection.create_item(label, attributes, password, replace=True)
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/secretstorage/collection.py", line 118, in create_item
    self.session = open_session(self.connection)
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/secretstorage/util.py", line 80, in open_session
    output, result = service.call(
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/secretstorage/util.py", line 63, in call
    return self.send_and_get_reply(msg)
  File "/home/reimar/mambaforge/envs/mssdev/lib/python3.10/site-packages/secretstorage/util.py", line 48, in send_and_get_reply
    raise DBusErrorResponse(resp_msg)
jeepney.wrappers.DBusErrorResponse: [org.freedesktop.DBus.Error.InvalidArgs] ('Client public key size is invalid',)

```

**Comment by ReimarBauer** [comment]
solved in 9.0

**Comment by ReimarBauer** [comment]
Duplicate of #1946
